AnaProd/tasks.py

```diff
@@ -35,7 +35,7 @@ def run(self):
             inDir = os.path.join(self.central_nanoAOD_path(), sample_name)
             os.makedirs(os.path.dirname(self.output().path), exist_ok=True)
             sh_call(['python3', producer, '--config', self.sample_config, '--inDir', inDir,
-                    '--outFile', self.output().path], env=self.cmssw_env())
+                    '--outFile', self.output().path, '--customisations', load_customisations()], env=self.cmssw_env())
```

Comment:
there is no such method. add customisation as luigi.Parameter and use it

Reply:
ok